alerting/build.gradle

```diff
@@ -81,8 +81,10 @@ dependencies {
 }
 
 javadoc.enabled = false // turn off javadoc as it barfs on Kotlin code
-licenseHeaders.enabled = true
+licenseHeaders.enabled = false
```

Comment:
Was this breaking something? Or why did we change to false here, seems like licenseHeaders check won't be run now?

Reply:
Did this change from job-schedular plugin change. I guess we don't need it. will check and change accordingly.